### DISCUSSION TITLE
UI live-coding dev server (proxy mode) config for NextJS ?

### DISCUSSION THREAD

**Discussion by pratapbunga** [quarkiverse/quarkus-quinoa#255] [Q&A] [answered]
Hi,

I'm a Frontend dev currently working to serve NextJS as part of Quarkus-Quinoa extension. I've found some config that is mentioned for Create-react-app and ViteJS in [docs](https://quarkiverse.github.io/quarkiverse-docs/quarkus-quinoa/dev/index.html#dev-server) to support HMR.

Wondering if there is anything that can be done for NextJS (v12)? Currently I'm getting this error `java.lang.RuntimeException: Quinoa package manager live coding port 3000 is still not listening after the checkTimeout.
`. Any help would be appreciated, thanks!

**Comment by melloware** [comment, marked as answer]
NextJS now works fine and is auto-detected in Quinoa 2.1.0+
